src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -123,17 +123,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     String user = userService.getCurrentUser().getEmail();
 
+
+
     // basicWithImages allows a, b, blockquote, br, cite, code, dd, dl, dt, em, i, li, ol,
     // p, pre, q, small, span, strike, strong, sub, sup, u, ul, and image tags
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.basicWithImages());
-    String recipient = request.getParameter("recipient");
+    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
```

Comment:
instead of `Whitelist.none()` we need `Whitelist.basicWithImages()`. this is what elli added to allow us styling of text.

Reply:
<img width="851" alt="Screen Shot 2019-04-04 at 1 25 24 PM" src="https://user-images.githubusercontent.com/33578456/55575953-136ab700-56de-11e9-9ccd-25c51ffce5ca.png">

Reply:
<img width="814" alt="Screen Shot 2019-04-04 at 1 31 28 PM" src="https://user-images.githubusercontent.com/33578456/55575965-1d8cb580-56de-11e9-8c4c-40d99e8bd559.png">